ENCODE_release.py

```diff
@@ -348,6 +359,61 @@ def has_inactive_pipeline(self, file_object):
                 return [p['@id'] for p in pipelines]
         return False
 
+    def _grab_bio_id(self, field, data):
+        """
+        Return biosample_term_id of embedded object.
+        """
+        return encodedcc.get_ENCODE(data.get(field),
+                                    self.connection).get('biosample_term_id')
+
+    def _get_associated_term_id(self, name, data):
+        """
+        Find biosample_term_id associated with particular object.
+        """
+        if name not in self.hela_associated_objects:
+            raise ValueError('Can only get biosample_term_id of known '
```

Comment:
_get_associated_term_id is only called
```
if name in self.hela_associated_objects
```
so this ValueError can't happen unless this function is called outside of 'associated_with_hela_data'.  But then self.hela_associated_objects won't be instantiated unless 'associated_with_hela_data'.